satinfo/amsua_aqua/readme

```diff
@@ -1,4 +1,8 @@
-aqua  20021012 000000  20071102 120000  amsua  10        4 5 6   8 9 10 11 12 13 14
-aqua  20071102 180000  20160923 120000  amsua   8            6   8 9 10 11 12 13 14                          # ch4 noisy; QC affected ; turn off ch1-5
-aqua  20160923 180000  21001231 240000  amsua   0                                                            # chs1-2 detector failure, cannot QC
-# NCEP continues to assimilate 8-13
+# launched 20020504, dumps started 20040413 ('Centerfield-of-view'),
+# 20051018 ('Warmest/clearest field-of-view',i.e. airswm)
+# 20070221 ('Every field-of-view' dumps start)
+# chan 14 used with no bias correction (iuse=4)
+# chs 1-3,15 assimilated by NOAA, but not NASA
+aqua  20021012 000000  20071102 120000  amsua  14 1 2 3 4 5 6   8 9 10 11 12 13 14 15
+aqua  20071102 180000  20160923 120000  amsua   8           6   8 9 10 11 12 13 14    # ch4 noisy; QC affected ; turn off ch1-5,15
+aqua  20160923 180000  21001231 240000  amsua   0                                     # chs1-2 detector failure, turn off
```

Comment:
Jeff. did you add a file that turns the satellite off? I am not seeing it in this PR. Maybe i am missing something.

Reply:
good catch - fixed now